src/main/java/org/jalasoft/moi/controller/services/FileService.java

```
@@ -68,8 +75,9 @@ public FileCode getFileById(Long id) {
     public FileCode addNewFile(String name, String code, Long projectID) throws IOException {
         FileCode newFilecode = new FileCode();
         newFilecode.setName(name);
-        newFilecode.setCode(saveFile(name,code, projectID));
-        newFilecode.setProject(projectRepository.findById(projectID).get());
+        newFilecode.setProject(projectRepository.findById(projectID).orElse(null));
```

Please don't save nulls, try using try / catch to communicate what happens, because when you save the `null` in the database you are **hiding the error**, and when someone tries to get this value he won't know why it is null.

---

I already solve that, thanks